hot3d/clips/bop_format_converters/hot3d_clips_to_bop_scenewise.py

```diff
@@ -0,0 +1,329 @@
+"""
```

Comment:
All files must have a header related to licensing of the project
```
# Copyright (c) Meta Platforms, Inc. and affiliates.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
I added the license. I also fixed an existing problem with the generation of out-of-frame-scope masks.

Reply:
Thank you @AnasIbrahim, I will fix the linting issue.